drivers/include/drivers/UnbufferedSerial.h

```diff
@@ -143,6 +143,35 @@ class UnbufferedSerial:
         return 0;
     }
 
+    /** Enable or disable input
+     *
+     * Control enabling of device for input. This is primarily intended
+     * for temporary power-saving; the overall ability of the device to operate
+     * for input and/or output may be fixed at creation time, but this call can
+     * allow input to be temporarily disabled to permit power saving without
+     * losing device state.
+     *
+     *  @param enabled      true to enable input, false to disable.
+     *
+     *  @return             0 on success
+     *  @return             Negative error code on failure
```

Comment:
That is not actually true...

Reply:
Ha! There is a subtle nuance in my comment (**"actually"** as in as existing **now**).

Reply:
Heh. But by definition the call does have to return a negative error code on failure. It's overriding the `FileHandle` API, so it has to retain that. So the statement is true.

But it is *also* possible that it never fails, so the "negative on failure" statement is true but redundant.